notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe explain why we even want to do this - why not just sort the entire recording all at once? </p><p> </p><p>I would also provide an overview of the entire process before you actually show them step by step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Maybe we can chat more about this. </p>

Reply:
<p>Could be nice to show the relational diagram in addition to some explanatory overview text</p>